src/plugins/intel_gpu/src/graph/impls/ocl/softmax.cpp

```diff
@@ -59,7 +59,6 @@ struct softmax_impl : typed_primitive_impl_ocl<softmax> {
     void update_dispatch_data(const kernel_impl_params& impl_param) override {
         auto kernel_params = get_kernel_params(impl_param, true);
         (_kernel_data.update_dispatch_data_func)(kernel_params.first, _kernel_data);
-        update_kernels_list_to_skip();
```

Comment:
(Random spot) Why these are removed? It should be updated at runtime whether to skip or not for shape agnostic kernel.

Reply:
Updating skip_execution in update_dispatch_data_func is not equivalent to updating after it?